### PR TITLE
Functionality working

### PR DESCRIPTION
**Pull request by rushi444**
Next things to do would be testing and pagination

### CODE REVIEW

File: src/redux/reducer.ts [hunk 1]
```diff
@@ -1,6 +1,47 @@
...
+import { GET_DEFAULT_IMAGES, SEARCH_BREED, SEARCH_BREED_ERROR, TOGGLE_LIKE } from './actions'
+
+export interface DefaultImages {
+  message: string[]
+  status: string
+}
+
+export interface InitialState {
+  images: DefaultImages | any
+  liked: string[] | any
```
Comment (funador) [inline src/redux/reducer.ts]: Can you think of a way to not use `any` here?

File: src/components/SearchResults.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <>
+      {error && <div>{error}</div>}
+      <ImageContainer>
+        {images?.message.map(
+          (image: string, index: number) =>
+            index < 12 && <DogImage imageLink={image} key={index} />,
+        )}
+      </ImageContainer>
+    </>
```
Comment (funador) [inline src/components/SearchResults.tsx]: Can you think of a way to filter this data before it reaches the component level?

**Review by funador — COMMENTED** [review]
Hi Rushi!

Thanks for sending this through. The code looks very reasonable and well thought out. Unfortunately, I couldn't get the app to run so I could not test things from that angle (no errors in the console, it is just not rendering the content). Let me know if you have a chance to look at that and I am happy to take another pass at it. (screen shot attached).  I also left a couple of comments in the PR. 

Looking forward to the chat tomorrow.  Cheers. 
Screen Shot 2020-01-20 at 11 37 48 AM

**Comment by funador** [comment]
> Hi Rushi!
> 
> Thanks for sending this through. The code looks very reasonable and well thought out. Unfortunately, I couldn't get the app to run so I could not test things from that angle (no errors in the console, it is just not rendering the content). Let me know if you have a chance to look at that and I am happy to take another pass at it. (screen shot attached). I also left a couple of comments in the PR.
> 
> Looking forward to the chat tomorrow. Cheers.
> Screen Shot 2020-01-20 at 11 37 48 AM

nvm, forgot to check out out your branch :) looks good

File: src/redux/reducer.ts (hunk 1 above)
Comment (rushi444) [reply src/redux/reducer.ts]: Oops, added the any in there for debugging, will remove them

**Comment by rushi444** [comment]
Hi, thanks for taking a look at my code. Just made the changes, look forward to speaking with you tomorrow!

**Comment by funador** [comment]
Hangout:

 To join the video meeting, click this link:
https://meet.google.com/daf-vyry-dju
Otherwise, to join by phone, dial +1 218-301-8448 and enter this PIN: 659
132 210#
To view more phone numbers, click this link:
https://tel.meet/daf-vyry-dju?hl=en&hs=5

**Pull request merged**